Pipeline/pipeline.py

```diff
@@ -62,7 +50,54 @@ def HTML_formater(df, name):
         f.write(html_string.format(table=df.to_html(classes="mystyle")))
 
 
-class pipeline:
+class Pipeline:
+    """
+
+    Parameters
+    ----------
+    df : pd.Dataframe
+
+    Examples
+    --------
+    >>> import pandas as pd
+    >>> import rossml as rsml
+
+    Importing and collecting data
+    >>> df = pd.read_csv('seal_fake.csv')
+    >>> df_val= pd.read_csv('xllaby_data-componentes.csv')

```

Comment:
I was thinking in add a new folder "example" to add some .csv files to use as example in docstring, etc, just like it's done in ROSS. Furthemore, we still have to do some changes in this example. It's too big and could take much time to run, when we have Travis and AppVeyor.

Reply:
seal_fake.csv is a expanded dataset with 50k points and xllaby_data-components.csv is a dataset with 4k points from XLTRC

Reply:
"I thought we only needed one file with the data, and the code would take care of splitting that in training and validation data." This is done in Pipeline. It will be clear when I finish the README

Reply:
To me it is not clear how to generate the `seal_fake.csv` file. Maybe we should have this information on the documentation or maybe the expanded dataset could be generated automatically by the code without the user having to provide the file path (don't know if that is possible).